hieradata/vagrant/common.yaml

```diff
@@ -348,6 +353,9 @@ profile::jenkinscontroller::jcasc:
     disabled: false
   datadog:
     host: "vagrant.local"
+    targetHost: "172.18.0.1" # docker0 interface in vagrant is non standard (because docker in docker)
+    collectBuildLogs: true
+    emitConfigChangeEvents: true
```

Comment:
Found https://javadoc.jenkins.io/plugin/datadog/index.html?org/datadog/jenkins/plugins/datadog/DatadogGlobalConfiguration.html but not sure about what this parameter is about

Reply:
Related to
> YAML JCasc bgufix for datadog binding field

?

Reply:
> This setting is useful if we want to track the config change history (like the configHistory plugin used to do with local git repositories) : it gives visibility on "what is happening" inside Jenkins.

Shouldn't this change be in a distinct PR?

Reply:
> Shouldn't this change be in a distinct PR?

IF the PR would change the current production setup: yes it should be a distinct PR.

But it is not the case, due to the way JCasc works for this particular Jenkins plugin (e.g. once you have set up through the UI, if JCasc does NOT change the value, then the value persists).

Not including this change in the PR here would change the setup once applied.

Reply:
Additionally, avoiding too much ci.jenkins restart is a constraint: it adds another compelling reason to have an all in one PR (since the scope is clearly defined)